examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -1459,7 +1459,7 @@ server cluster GeneralDiagnostics = 51 {
 server cluster GroupKeyManagement = 63 {
   enum GroupKeySecurityPolicy : ENUM8 {
     kStandard = 0;
-    kLowLatency = 1;
+    kTrustFirst = 1;
   }
```

Comment:
Order is changing as well, see:

https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/4890/files#diff-aac62f21cb838795deb17839377022a01f9f4358583125025b776badb0038b17

the PR has not merged yet, so I will make sure to merge

Reply:
Will change value in a follow up PR